index.js

```diff
@@ -42,12 +42,19 @@ module.exports.pitch = function(request) {
 		if(err) return callback(err);
 		if (entries[0]) {
 			var workerFile = entries[0].files[0];
-			var constructor = "new Worker(__webpack_public_path__ + " + JSON.stringify(workerFile) + ")";
-			if(query.inline) {
-				constructor = "require(" + JSON.stringify("!!" + path.join(__dirname, "createInlineWorker.js")) + ")(" +
+			var constructor
+			if(query.service) {
+				constructor = "('serviceWorker' in navigator)" +
+				"? navigator.serviceWorker.register(__webpack_public_path__ + " + JSON.stringify(workerFile) + ", options);" +

```

Comment:
I'm getting a syntax error in the generated code on this line, caused by the semi-colon before the ternary operator colon:

If you remove it, it seems to work correctly. 

```
? navigator.serviceWorker.register(__webpack_public_path__ + " + JSON.stringify(workerFile) + ", options) " 
```